pkg/ctl/delete/cluster.go

```diff
@@ -52,7 +52,15 @@ func doDeleteCluster(cmd *cmdutils.Cmd, force bool) error {
 
 	ctl, err := cmd.NewProviderForExistingCluster()
 	if err != nil {
-		return err
+		if !force {
+			return err
+		}
+		// initialise the controller without refreshing the cluster status.
```

Comment:
work adding a log about skipping the error?

Reply:
👍 will add one